packages/core/src/transport.ts

```diff
@@ -68,6 +69,19 @@ export class Batch<T> {
     }
   }
 
+  sizeInBytes(candidate: string) {
+    // Accurate byte size computations can degrade performances when there is a lot of events to process
+    if (!HAS_MULTI_BYTES_CHARACTERS.test(candidate)) {
+      return candidate.length
+    }
+
+    if (isIE()) {
```

Comment:
⚠️  isIE is only used for unit test and must not be used in prod code because it relies on simple UA matching.
You should use feature detection instead.